EIPS/eip-5851.md

```diff
@@ -0,0 +1,256 @@
+
+### Definitions
+
+- ZKP: stands for Zero-knowledge Proof which means that a prover can convince a verifier that an assertion is correct without providing any useful information to the verifier.
+
+- SBT: Known as Soulbound tokens, these are non-fungible and non-transferrable tokens that is used for defining the identity of the users.
+
+- SBT Certificates: SBT that represent the ownerships of ID signatures corresponding to the claims defined in `function standardClaim()`.
+
+- Verifiable credentials (VC): A Verifiable Credential is a collection of claims made by an issuer. These are temper evident credentials that allow the holders to prove that they posses certain characteristics (for example, passport verification, constraints like value of tokens in your wallet, etc) as demanded by the verifier entity.
+
+- Claims: Decentralised Identity(DID) Claims are a set of assertions that the DID Holder must fulfill to be verified.
+
+- Holder: The holder is the entity that stores the claim, such as a digital identity provider or a DID registry. The holder is responsible for validating the claim and providing verifiable evidence of the claim.
+
+- Claimer: A claimer is the party making a claim, such as in an identity verification process.
+
+- Issuer: An entity creates a verifiable credential from claims about one or more subjects to a holder. Example issuers include governments, corporations, non-profit organizations, trade associations, and individuals.
+
+- Verifier: A verifier in DID is an entity that validates data provided by an issuer of verifiable credentials, determining its accuracy, origin, currency and trustworthiness.
+
+
+
+### Metadata Standard
+
+Claims MUST be stored in the following structures:
+
+#### 1. Metadata information
+
+Each claim requirement MUST be exposed using the following struct:
+
+```solidity
+    /** Metadata
+    * 
+    * @param title defines the name of the claim field
+    * @param kind is the nature of the data (bool,string,address,bytes,..)
+    * @param description additional information about claim details.
+     */
+    struct Metadata {
+        string title;
+        string kind;
+        string description;
+    }
+```
+
+### 2. Values Information
+
+This structure will be used to define the actual claim information, based on the description of the `Metadata` structure, the structure is similar to `Values` structure of [EIP-3475](./eip-3475.md).
+
+### 3. Claim structure
+
+Claims (eg. `age >= 18`, jurisdiction in allowlist, etc.) are represented by one or many instances of the `Claim` structure below:
+
+```solidity
+    /** Claims
+    * 
+    * Claims structure consist of the conditions and value that holder claims to associate and verifier has to validate them.
+    * @notice the below given parameters are for reference purposes only, developers can optimize the fields that are needed to be represented on-chain by using schemes like TLV, encoding into base64 etc.
```

Comment:
These two sentences seem to be at odds:

> Claims [...] are represented by one or many instances of the `Claim` structure below:

> the below given parameters are for reference purposes only

What is the `Claim` structure supposed to look like?

Reply:
`{
   "title":"age",
   "type":"unit",
   "description":"age of the person based on the birth date on the legal document",
   "logic":">=",
   "value":"18"
}`